### DISCUSSION TITLE
Thank you leapful ❤️

### DISCUSSION THREAD

**Discussion by bluebill1049** [react-hook-form/react-hook-form#9249] [Show and tell]
https://react-hook-form.com/about-us

![Screen Shot 2022-10-20 at 1 32 38 pm](https://user-images.githubusercontent.com/10513364/196842463-ce331c50-f1cd-42ea-9212-54b8089b2458.png)

Thanks @leapful for the hidden work that you have been helping everyone in the community. (Please send a PR to adjust if you wish)

**Comment by Moshyfawn** [comment]
Let's go! Appreciate the work you're doing, @leapful!

**Comment by leapful** [comment]
Thank you 😄
